### DISCUSSION TITLE
Add publication cost metadata

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#60] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

It would generate a transparency about the money spend for publications and would allow third parties to analyse the costs of publications, see http://opencost.ur.de/
I strongly believe, that payments are part of the metadata of a a publication and are missing in known metadate schemes. They are  very often strongly related to other metadata fields like licensing or funding. 
This information can be valluable for publishers (to send information to their customer), research institutions (to provide open information about their spending for publishing), and for funders (to collect data about publishing costs)

### What solution might meet your needs?

Adding Metadata about costs associated with publishing

### What alternatives have you tried or considered?

We have already proposed a matadata scheme for payments of article fees (see https://github.com/opencost-de/opencost ) which was discussed in an international workshop and online. Information about transformative agreements will  be added soon. 
We use this scheme to send data via OAI e.g. to OpenAPC, see https://openapc.net/

### Is there anything else you would like to share?

I gave already a talk about this issue and you can find a possible implementation at https://doi.org/10.5281/zenodo.7993722 , slide 39
I'm happy to talk with you about this issue.

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
Perhaps this idea could be simplified in structure and expanded in concept to include access cost. Schema.org supports this in the isAccessibleForFree and similar fields, but DataCite has no support for this yet.

**Comment by aw-bib** [comment]
As we worked on this issue in _real code_ on the openCost side to comply with some funder requirement and one of our own open issues (https://github.com/opencost-de/opencost/issues/67) I added out current approach as PR against the DataCite code for v4.7 via https://github.com/datacite/schema/pull/159
